scripts/performance/common.py

```diff
@@ -180,9 +181,20 @@ def run(self, working_directory: str = None) -> None:
                                 getLogger().info(line)
 
                     proc.wait()
+                    return (proc.returncode, quoted_cmdline)
+
+
+    def run(self, working_directory: str = None) -> None:
+        '''Executes specified shell command.'''
 
-                    if proc.returncode not in self.success_exit_codes:
-                        getLogger().error(
-                            "Process exited with status %s", proc.returncode)
-                        raise CalledProcessError(
-                            proc.returncode, quoted_cmdline)
+        retrycount = 0
+        (returncode, quoted_cmdline) = self.__runinternal(working_directory)
+        while returncode not in self.success_exit_codes and retrycount < self.__retry:
+            (returncode, _) = self.__runinternal(working_directory)
+            retrycount += 1
+
+        if returncode not in self.success_exit_codes:
+            getLogger().error(
+                "Process exited with status %s", returncode)
+            raise CalledProcessError(
```

Comment:
nit: remove the line break?

Reply:
Actually there are many line breaks in this file look weird to me and have been there before your change. Not sure if that's intentional but feel free to keep/change all of them.

Reply:
Yeah I didn't actually add these line breaks. It just looks that way because of how the diff winds up.